### PR TITLE
added navbar component

### LINKED ISSUE #27: Add Navbar Component
**Create a Navbar for Header in the component file**
Design Should be the Same as shown on [HomePage](https://github.com/theupweb/idea-lab#home-page-) (Check Readme File)

Framework used: Tailwind CSS
Tech stack: HTML, CSS, js, React

### PR DESCRIPTION
**Pull request by shrushti2000**
Fixes #27 
Added Navbar Component
idealab-navbar

### CODE REVIEW

**Comment by ghost** [comment]
Great work @shrushti2000! 🥇

File: src/Components/Home/Nav.js [hunk 1]
```diff
@@ -1,9 +1,127 @@
...
+import Logo from "../../Logo";
 
 export default function Nav() {
-    return (
-        <div>
-            Navbar
+  
+  return (
+    <div class="relative bg-white">
+      <div class="max-w-7xl mx-auto px-4 sm:px-6">
```
Comment (sanjay270899) [inline src/Components/Home/Nav.js:8]: We should use `className` instead of `class` in React.

File: src/Components/Home/Nav.js [hunk 2]
```diff
@@ -1,9 +1,127 @@
...
+import React from "react";
+import Logo from "../../Logo";
 
 export default function Nav() {
-    return (
-        <div>
-            Navbar
+  
+  return (
+    <div class="relative bg-white">
```
Comment (sanjay270899) [inline src/Components/Home/Nav.js:7]: @shrushti2000 Header background should be transparent. 

Now:
image

My Suggestion:
image

File: src/App.js [hunk 3]
```diff
@@ -2,10 +2,11 @@
 import './App.css';
 import Logo from './Logo.js';
```
Comment (sanjay270899) [inline src/App.js:3]: Remove this `Logo` import.

File: src/Components/Home/Nav.js [hunk 4]
```diff
@@ -1,9 +1,127 @@
...
+          <div class="pt-5 pb-6 px-5">
+            <div class="flex items-center justify-between">
+              <div>
+                <Logo />
+              </div>
+              <div class="-mr-2">
+                <button
+                  type="button"
+                  class="bg-white rounded-md p-2 inline-flex items-center justify-center text-gray-400 hover:text-gray-500 hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-inset focus:ring-indigo-500"
+
```
Comment (sanjay270899) [inline src/Components/Home/Nav.js:62]: How to close nav-bar in mobile?
image
**Close** button is not working.

**Comment by sanjay270899** [comment]
@JoshuaPoddoku Should I fix these changes?

**Pull request merged**